composites/Plugin/Shared/components/YoastModal.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Modal from "react-modal";
+import styled from "styled-components";
+
+import colors from "../../../../style-guide/colors.json";
+import SvgIcon from "./SvgIcon";
+
+const StyledHeading = styled.h1`
+	float: left;
+	margin: -4px 0 2rem;
+	font-size: 1rem;
+`;
+
+const StyledButton = styled.button`
```

Comment:
Could this be named a bit more specific to the function it has, since it is specifically for the x-cross button?

Reply:
Is there no reusable component for this?

Reply:
As far as I see, no. In the notifications there's a dismiss button with `<StyledIcon icon="times" ...` not reusable and with different styling.